java/api/src/main/java/org/ray/api/PyActorCall.java

```diff
@@ -0,0 +1,41 @@
+// Generated by `RayCallGenerator.java`. DO NOT EDIT.
+
+package org.ray.api;
+
+/**
+ * This class provides type-safe interfaces for remote actor calls.
+ **/
+@SuppressWarnings({"rawtypes", "unchecked"})
+interface PyActorCall<A> {
```

Comment:
same here, `A`  isn't needed.

Reply:
The generic type A is useless for PyActorCall, I have removed it.